[2025-05-13T22:01:18.683+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-13T22:01:18.727+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.model_data scheduled__2025-05-09T00:00:00+00:00 [queued]>
[2025-05-13T22:01:18.750+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.model_data scheduled__2025-05-09T00:00:00+00:00 [queued]>
[2025-05-13T22:01:18.751+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-13T22:01:18.785+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): model_data> on 2025-05-09 00:00:00+00:00
[2025-05-13T22:01:18.816+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=295) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-13T22:01:18.822+0000] {standard_task_runner.py:72} INFO - Started process 310 to run task
[2025-05-13T22:01:18.823+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'model_data', 'scheduled__2025-05-09T00:00:00+00:00', '--job-id', '1001', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmptlcjdj5d']
[2025-05-13T22:01:18.826+0000] {standard_task_runner.py:105} INFO - Job 1001: Subtask model_data
[2025-05-13T22:01:18.915+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.model_data scheduled__2025-05-09T00:00:00+00:00 [running]> on host ff2d23694728
[2025-05-13T22:01:19.095+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='elkhalfimaryame@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='model_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-09T00:00:00+00:00'
[2025-05-13T22:01:19.100+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-13T22:01:19.101+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-13T22:01:19.102+0000] {logging_mixin.py:190} INFO - Current task name:model_data state:running start_date:2025-05-13 22:01:18.729017+00:00
[2025-05-13T22:01:19.103+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-13T22:01:19.104+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-13T22:01:19.121+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-13T22:01:19.243+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-13T22:01:19.261+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-13T22:01:19.263+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=model_data, run_id=scheduled__2025-05-09T00:00:00+00:00, execution_date=20250509T000000, start_date=20250513T220118, end_date=20250513T220119
[2025-05-13T22:01:19.304+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-13T22:01:19.308+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-13T22:01:19.312+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-13 21:58:11.465605+00:00
[2025-05-13T22:01:19.315+0000] {logging_mixin.py:190} INFO - Task hostname:ff2d23694728 operator:PythonOperator
[2025-05-13T22:01:19.364+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-13T22:01:19.426+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-13T22:01:19.448+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
